bots/pr/src/main/java/org/openjdk/skara/bots/pr/BackportCommand.java

```diff
@@ -104,7 +106,14 @@ public void handle(PullRequestBot bot, HostedCommit commit, CensusInstance censu
 
         try {
             var hash = commit.hash();
-            var fork = bot.writeableForkOf(targetRepo);
+            var optionalFork = bot.writeableForkOf(targetRepo);
+            if (optionalFork.isEmpty()) {
+                reply.print("@" + username + " [" + repoName + "](" + targetRepo.webUrl() + ") is not a valid target for backports. ");
+                reply.print("List of valid repositories: ");
+                reply.println(String.join(", ", bot.forkRepoNames()));
```

Comment:
Should this be further filtered to only list repos that have the same "base" (share some commits) as the source repo? For example, if we some day configure one of the `jfx` or `jmc` update release repos to allow backport APIs, you wouldn't want to target it from a commit in the `jdk` repo, so listing those repos could be confusing. This could be done as a follow-up, since it is only a hypothetical problem for now.

Reply:
I agree that it would be much better to filter the list of repos, but I'm not sure how to easily implement this. I figured this list was easily available and better than nothing.